server/http.go

```diff
@@ -349,10 +566,67 @@ func (p *Plugin) postMeeting(creator *model.User, meetingID int, channelID strin
 	if appErr = p.storeMeetingPostID(meetingID, createdPost.Id); appErr != nil {
 		p.API.LogDebug("failed to store post id", "err", appErr)
 	}
-
 	return nil
 }
 
+func (p *Plugin) askUserPMIMeeting(userID string, channelID string) {
+	apiEndPoint := fmt.Sprintf(APIToAskForPMI, manifest.ID)
+
+	slackAttachment := model.SlackAttachment{
+		Pretext: AskForPMIMeeting,
+		Actions: []*model.PostAction{
+			{
+				Id:    "WithPMI",
+				Name:  UsePersonalMeetingID,
+				Type:  "button",
+				Style: "primary",
+				Integration: &model.PostActionIntegration{
+					URL: apiEndPoint,
+					Context: map[string]interface{}{
+						Action:    UsePersonalMeetingID,
+						UserID:    userID,
+						ChannelID: channelID,
+					},
+				},
+			},
+			{
+				Id:    "WithoutPMI",
+				Name:  UseAUniqueMeetingID,
+				Type:  "button",
+				Style: "primary",
+				Integration: &model.PostActionIntegration{
+					URL: apiEndPoint,
+					Context: map[string]interface{}{
+						Action:    UseAUniqueMeetingID,
+						UserID:    userID,
+						ChannelID: channelID,
+					},
+				},
+			},
+		},
+	}
+
+	post := &model.Post{
+		ChannelId: channelID,
+		UserId:    p.botUserID,
+	}
+	model.ParseSlackAttachment(post, []*model.SlackAttachment{&slackAttachment})
+	p.API.SendEphemeralPost(userID, post)
+}
+
+func (p *Plugin) getPMISettingData(userID string) (string, error) {
+	if preferences, reqErr := p.API.GetPreferencesForUser(userID); reqErr == nil {
```

Comment:
```suggestion
	if preferences, reqErr := p.API.GetPreferencesForUser(userID); reqErr != nil {
```
Reverse the condition.